homeassistant/components/unifi_access/config_flow.py

```diff
@@ -0,0 +1,69 @@
+"""Config flow for the UniFi Access integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from uiaccessclient import ApiClient, SpaceApi
+from uiaccessclient.openapi.exceptions import ForbiddenException, UnauthorizedException
+import urllib3.exceptions
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_API_TOKEN, CONF_HOST
+from homeassistant.core import HomeAssistant
+
+from .const import DEFAULT_HOSTNAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST, default=DEFAULT_HOSTNAME): str,
+        vol.Required(CONF_API_TOKEN): str,
+    }
+)
+
+
+class UniFiAccessConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for UniFi Access."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Process configuration form."""
+        errors: dict[str, str] = {}
+
+        if user_input is not None:
+            self._async_abort_entries_match({CONF_HOST: user_input[CONF_HOST]})
+
+            await _validate_input(self.hass, user_input, errors)
+            if not errors:
+                return self.async_create_entry(title="UniFi Access", data=user_input)
```

Comment:
Do we have something unique we can use to make the config entry even more uniuqe?